spec.html

```diff
@@ -34631,7 +34486,7 @@ <h1>
               1. Let _patternCharacters_ be a List whose elements are the code unit elements of _P_.
             1. Let _parseResult_ be ParsePattern(_patternText_, _u_).
             1. If _parseResult_ is a non-empty List of *SyntaxError* objects, throw a *SyntaxError* exception.
-            1. Assert: _parseResult_ is a Parse Node for |Pattern|.
+            1. Assert: _parseResult_ is a |Pattern|.
```

Comment:
```suggestion
            1. Assert: _parseResult_ is a |Pattern| Parse Node.
```

right?

Reply:
Sure. I think there's other usages like this though. It'd be nice to normalise them.

Reply:
I don't think minimising churn in this case is the top priority. We should do whatever reads clearest. Maybe it's worth discussing in the editor call.

Reply:
Discussion sounds good.

Reply:
Editor call: We should normalise these 3 forms to "a |Foo| Parse Node".